oauth-sso/sso-authorization-server/pom.xml

```diff
@@ -100,6 +135,19 @@
 					</excludes>
 				</configuration>
 			</plugin>
+			<plugin>
```

Comment:
Could you double-checked it's needed?

Reply:
Plugin is not required and hence removed.
Thanks